examples/payload-processor/Makefile

```
@@ -0,0 +1,59 @@
+.PHONY: build clean test
+
+# Detect cluster architecture if not specified
+ARCH ?= $(shell kubectl get nodes -o jsonpath='{.items[0].status.nodeInfo.architecture}' 2>/dev/null || echo "amd64")
+
+# Container runtime (podman or docker)
+CONTAINER_RUNTIME ?= $(shell command -v podman 2>/dev/null || command -v docker 2>/dev/null)
+
+# Build using container (for cross-platform builds from macOS)
+build:
```

checkmake (0.3.2)

[warning] Target body for "build" exceeds allowed length of 5 lines (8).

(maxbodylength)